public/content/documentation/native/controls/dropdown.md

```diff
@@ -0,0 +1,323 @@
+- The difference between a menu and dropdown is a menu item performs an action when activated.  A dropdown item only replaces the current option
+- The screen reader focus moves directly to first option in dropdown upon double tapping the dropdown button and is confined in the list.
+- Sometimes, a hidden "dismiss context menu" button after the last item is available to close it.
+- Focus should go back to the triggering dropdown button, displaying the new option
+- There must be a visible label for the dropdown field that is not a placeholder and it describes the purpose of the dropdown.
+- The screen reader focus also remains confined in the dropdown list
+- The state of expanded or collapsed should be announced
+
+### Name
+- Programmatic name describes the purpose of the control.
+- If visible text label exists, the programmatic name should match the visible text label.
+    - **Note:** Setting a programmatic name while a visible text label exists may cause VoiceOver to duplicate the announcement of the name. If this happens, hide the visible text label from VoiceOver recognition
+- When naming a button, do not add "button" to the programmatic name (label). Assigning "Button" as the role will handle this announcement.
+  - **Incorrect announcement:** "Submit button, Button"
+  - **Correct announcement:** "Submit, Button"
+- Placeholder or value text is NOT the programmatic name
+
+- **UIKit**
+  - You can programmatically set the visible label with `setTitle()`.
+    - The button’s title will overwrite the button’s `accessibilityLabel`.
+  - If a visible label is not applicable in this case, set the button's `accessibilityLabel` to the label of your choice.
+    - To do this in Interface Builder, set the label using the Identity Inspector
+  - To hide labels from VoiceOver programmatically, set the label's `isAccessibilityElement` property to `false`
+  - To hide labels from VoiceOver using Interface Builder, uncheck `Accessibility Enabled` in the Identity Inspector.
+- **SwiftUI**
+  - If no visible label, use view modifier `accessibilityLabel(_:)`.
+  - If button has icon(s), hide the icon(s) from VoiceOver by using view modifier `accessibilityHidden(true)`.
+
+### Role
+- When using non-native controls (custom controls), roles will need to be manually coded.
+- One option is to use a table view and a button inside a UIStackView
+
+- **UIKit**
+  - Use `UIButton`
+  - If necessary, set `accessibilityTraits` to `.button`.
+- **SwiftUI**
+  - Use native `Button` view
+  - If necessary, use view modifier `accessibilityAddTraits(.isButton)` to assign the role as Button.
+  - If applicable, use view modifier `accessibilityRemoveTraits(:)` to remove unwanted traits.  
+
+### Groupings
+- Group visible label with button, if applicable, to provide a programmatic name for the button.
+
+- **UIKit**
+  - Ensure that the child elements of the overarching view you want to group in has their `isAccessibilityElement` properties set to false.
```

Comment:
The code looks correct, for the indention pattern under Groupings > UI Kit, but the display is off. Maybe try another `parse-md-files`?